spec/models/creation/repository_spec.rb

```diff
@@ -47,8 +47,9 @@
 
   context "#search_with" do
     let(:cake_category) { create(:category) }
-    let!(:cake) { create(:cake, category: cake_category) }
-    let!(:cookie) { create(:cake) }
+    let(:cookie_category) { create(:category) }
+    let!(:cake) { create(:cake, name: 'cake', category: cake_category) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.